### PR TITLE
Feat: Screens for Notes App

### PR DESCRIPTION
**Pull request by Rohitprasad83**
## Added all the screens
- Landing page
- Sign up
- Login
- Home
- Archive
- Labels
- Trash
- Profile

## Gif
chrome_doAmiTwLKm
chrome_AEvNQqfpM6

### CODE REVIEW

File: src/components/layout/Layout.jsx [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
 import { Sidebar } from '../sidebar/Sidebar.jsx'
-import { MainContent } from '../main/MainContent.jsx'
 import { useState, useEffect } from 'react'
-import './Home.css'
-function Home() {
+import { Link } from 'react-router-dom'
+function Layout() {
   const [openSidebar, setOpenSidebar] = useState(false)
   useEffect(() => {
     window.onresize = () =>
```
Comment (rohanmathur91) [inline src/components/layout/Layout.jsx:9]: suggestion: put a comment on what this code is doing

File: src/components/sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
+        </Link>
       </div>
       <div className="sidebar__items">
-        <i className="fa-solid fa-user"></i> Profile
+        <Link to="/profile">
+          <i className="fa-solid fa-user"></i> Profile
+        </Link>
       </div>
       <div className="sidebar__items ">
         <i className="fa-solid fa-filter input__icons"></i>
```
Comment (rohanmathur91) [inline src/components/sidebar/Sidebar.jsx:32]: have a array having this data, and just loop over on it, instead of repeating

**Review by hiteshverma27 — APPROVED** [review]
- Good naming convention followed 
- Readable code
- Good folder structure

File: src/pages/label/Label.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Layout } from '../../components/layout/Layout'
+import { MainContent } from '../../components/main/MainContent.jsx'
```
Comment (rohanmathur91) [inline src/pages/label/Label.jsx]: try barrel exports

**Review by rohanmathur91 — APPROVED** [review]
- Good use of CSS modules
- Code is well structured and UI also looks good
- Added few suggestions

File: src/components/layout/Layout.jsx (hunk 1 above)
Comment (Rohitprasad83) [reply src/components/layout/Layout.jsx:9]: Thanks added a comment for this

File: src/pages/label/Label.jsx (hunk 3 above)
Comment (Rohitprasad83) [reply src/pages/label/Label.jsx]: Thanks now using barrel plus absolute path

File: src/components/sidebar/Sidebar.jsx (hunk 2 above)
Comment (Rohitprasad83) [reply src/components/sidebar/Sidebar.jsx:32]: will do it later on

**Pull request merged**